src/identity.js

```diff
@@ -804,7 +804,8 @@ export class Identity extends EventEmitter {
                 teaser,
                 max_age: maxAge,
                 locale,
-                one_step_login: oneStepLogin || ''
+                one_step_login: oneStepLogin || '',
+                prompt: 'select_account'
```

Comment:
Shouldn't this be based on `siteSpecificLogout`: `siteSpecificLogout ? 'select_account' : ''`?
To avoid showing it for those that aren't using `siteSpecificLogout` yet.

Reply:
👍  good catch